_benchmark/configuring-benchmark.md

```diff
@@ -0,0 +1,202 @@
+| `datastore.secure` | Boolean | If set to `false`, OpenSearch assumes a HTTP connection. If set to true, it assumes a HTTPS connection. |
+| `datastore.ssl.verification_mode` | String | When set to the default `full`, the metric store’s SSL certificate is checked. To disable certificate verification set this value to `none`. |
+| `datastore.ssl.certificate_authorities` | String | Determines the path on the local file system to the certificate authority’s signing certificate.
+| `datastore.user` | Username |  Sets the name of the OpenSearch user for the metrics store. |
+| `datastore.password`: | String | Sets the password of the OpenSearch user for the metrics store. Alternatively, this password can be configured using the `OSB_DATASTORE_PASSWORD` environment variable, which avoids storing credentials in a plain text file. The environment variable will take precedence over the config file if both define a password. |
+| `datastore.probe.cluster_version` | String | Enables automatic detection of the metric store’s version. Default is `true`. |
+| `datastore.number_of_shards` | Integer | The number of primary shards that the `opensearch-*` indexes should have. Any updates to this setting after initial index creation will only be applied to new `opensearch-*` index. Default is the [OpenSearch static index value]({{site.url}}{{site.baseurl}}/api-reference/index-apis/create-index/#static-index-settings).
+| `datastore.number_of_replicas` | Integer |  The number of replicas each primary shard has. Any updates to this setting after initial index creation will only be applied to any new `opensearch-* `indexes. Default is the [OpenSearch static index value]({{site.url}}{{site.baseurl}}/api-reference/index-apis/create-index/#static-index-settings). |
+
+### Examples
+
+You can use the following examples to set reporting values in your own cluster.
+
+This example defines an unprotected metrics store in the local network:
+
+```
+[results_publishing]
+datastore.type = opensearch
+datastore.host = 192.168.10.17
+datastore.port = 9200
+datastore.secure = false
+datastore.user =
+datastore.password =
+```
+
+This example defines a secure connection to a metrics store in the local network with a self-signed certificate:
+
+```
+[results_publishing]
+datastore.type = opensearch
+datastore.host = 192.168.10.22
+datastore.port = 9200
+datastore.secure = true
+datastore.ssl.verification_mode = none
+datastore.user = user-name
+datastore.password = the-password-to-your-cluster
+```
+
+## workloads
+
+This section defines how workloads are retrieved. All keys are read by OpenSearch using the syntax `<<workload-repository-name>>.url`, which you can select using the OpenSearch Benchmark CLI  `--track-repository=workload-repository-name"` option. By default, OpenSearch chooses the track repository using the `default.url`, `https://github.com/opensearch-project/opensearch-benchmark-workloads`.
```

Comment:
1. `--track-repository` --> `--workload-repository`
2. `OpenSearch chooses the track repository` --> `workload repository`